lib/tether/tether.go

```diff
@@ -65,6 +65,7 @@ type tether struct {
 	sink extraconfig.DataSink
 
 	incoming chan os.Signal
+	done     chan bool
```

Comment:
If you just want to signal stop, use `chan struct{}`, and then close the channel when you want to signal.